yang/ietf-schedule.yang

```diff
@@ -744,6 +744,13 @@ module ietf-schedule {
       description
         "Indicates the schedule type.";
     }
+    leaf local-time {
+      type yang:date-and-time;
+      config false;
+      description
+        "Reports the local time as used by the entity that
+         hosts the schedule.";
```

Comment:
```suggestion
         hosts the schedule. This parameter is only reported if schedule is expressed in local time format.";
```
Note that 3231 requires calendar schedules are always expressed in local time. I think if the schedule is using UTC time, no need to know the local time info.
I guess I don't really understand what time is reported by this parameter? The time when this value is retrieved? or the time when the schedule is managed?

Reply:
You are right for calendar schedules, but other types do not have such restriction. 

This will report the actual local-time as seen by the entity. 

No need to restrict the attribute to non UTC but leave that the manager to infer whatever useful from the parameter.

Reply:
Sure, I think I can live with that.